src/Monitor/Monitor/OutputClasses/PSMetricAlertRuleV2.cs

```diff
@@ -62,7 +67,7 @@ public PSMetricAlertRuleV2(MetricAlertResource metricAlertResource)
         /// Gets or sets list of criteria.
         /// </summary>
         [JsonProperty(PropertyName = "criteria")]
-        public new IList<PSMetricCriteria> Criteria { get; set; }
+        public new IList<IPSMultiMetricCriteria> Criteria { get; set; }
```

Comment:
Potential Breaking Change: Property criteria PSMetricCriteria -> IPSMultiMetricCriteria (same as above, new type would have to have the same properties)